### PR TITLE
Documentation and bug fixes

### LINKED ISSUE #153: [PE-D] Error message is displayed incorrectly for invalid commands
how to reproduce this error:
1) start the jar file
2) repeatedly press enter

I am using cygwin terminal (not sure if this affects anything)

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: AndreWongZH/ped#1

### LINKED ISSUE #163: [PE-D] tag-e output from UG shows as not valid
Followed the exact output from UG for tag-e function but cant seem to get a valid output

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: BenardoTang/ped#3

### PR DESCRIPTION
**Pull request by Chongjx**
Completed:
Closes #153
Closes #163 

In progress:
#171

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -526,7 +529,7 @@ Adds a new item to the list of todo items.
 
 Format: `delete-e INDEX`
 
-* Deletes the event at the specified INDEX. The index refers to the index number shown in the displayed event list (list-e). The index must be a **positive integer** (1, 2, 3, …).
+- Deletes the event at the specified INDEX. The index refers to the index number shown in the displayed event list (list-e). The index must be a **positive integer** (1, 2, 3, …).
```
Comment (R-Ramana) [inline docs/UserGuide.md:532]: Why was this change made ah? I don't think there is a difference to the output right?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -56,6 +56,9 @@ Any points with the 💡 emoji represents an additional information. You can acc
 
 5. Use the command `help` to get a list of commands and their usages before using the application (if needed).
 
+💡 Supported terminals are: Windows 10 Command Prompt, Windows Powershell, macOS Terminal and linux. Other terminals, such as Cygwin may not support color display.
```
Comment (R-Ramana) [inline docs/UserGuide.md:59]:
```suggestion
💡 Supported terminals are: Windows 10 Command Prompt, Windows Powershell, macOS and Linux Terminals. Other terminals, such as Cygwin may not support color display.
```

**Review by R-Ramana — APPROVED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (Chongjx) [reply docs/UserGuide.md:532]: Just for standardization since all are using '-'

File: docs/UserGuide.md [hunk 3]
```diff
@@ -56,7 +56,7 @@ Any points with the 💡 emoji represents an additional information. You can acc
 
 5. Use the command `help` to get a list of commands and their usages before using the application (if needed).
 
-💡 Supported terminals are: Windows 10 Command Prompt, Windows Powershell, macOS Terminal and linux. Other terminals, such as Cygwin may not support color display.
+💡 Supported terminals are: Windows 10 Command Prompt, Windows Powershell, macOS Terminal and linux Terminal. Other terminals, such as Cygwin may not support color display.
```
Comment (R-Ramana) [inline docs/UserGuide.md:59]:
```suggestion
💡 Supported terminals are: Windows 10 Command Prompt, Windows Powershell, macOS Terminal, and Linux Terminal. Other terminals, such as Cygwin may not support color display.
```

**Review by R-Ramana — APPROVED** [review]

File: docs/UserGuide.md (hunk 3 above)
Comment (Chongjx) [reply docs/UserGuide.md:59]: I updated this. Will ask for re-review when I complete all changes.

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -81,13 +81,16 @@ The architecture design is given in the diagram above. The main components of No
...
-NotUS manages the flow of the application. On launch, it will create the necessary components, as listed above and then attempts to load any existing saved files into the application. Subsequently, it will accept and interpret the user input and execute the commands accordingly. The diagram below depicts the main flow of the application.
+NotUS manages the flow of the application. On launch, it will create the necessary components, as listed above and then attempts to load any existing saved files into the application. Subsequently, it will accept and interpret the user input and execute the commands accordingly. Figure 2 below depicts the main flow of the application.
 
 <p align="center">
   <img alt="NotUS" src="diagrams/out/Notus.png" />
   <br><em>Figure 2</em>
 </p>
 
+💡 Due to the limitation of 
+💡 The lifeline for Parser and Command should end at destroy marker (X) but due to a limitation of PlantUML, the lifeline reaches the end of diagram. This applies to the rest of the sequence diagrams in the document.
```
Comment (R-Ramana) [inline docs/DeveloperGuide.md:92]: Can verify this again? Looks off.

File: docs/UserGuide.md [hunk 5]
```diff
@@ -332,7 +332,7 @@ Create tags.
 Format: `create-t /tag TAG [TAG_COLOR] [/tag TAG]...`
 
 - Creates a tag with the name TAG. TAG can only be a single word.
-- [TAG_COLOR] is an optional input to specify the tag color to display. The possible colors are red, green, blue, yellow, purple, cyan, white. The tag color is set as white by default.
+- [TAG_COLOR] is an optional input to specify the tag color to display. The possible colors are red, green, blue, yellow, purple, cyan, white. Any other input will set the tag as white. If no color is specified, the color of the tag will be set to the default white color. This applies to other input that has `/tag TAG [TAG_COLOR]` as an optional field.
```
Comment (R-Ramana) [inline docs/UserGuide.md:335]:
```suggestion
- [TAG_COLOR] is an optional input to specify the tag color to display. The possible colors are red, green, blue, yellow, purple, cyan, white. If no color is specified, or an invalid input is entered, the color of the tag will be set to the default white color. This applies to all other inputs that have `/tag TAG [TAG_COLOR]` as an optional field as well. 
```

**Review by R-Ramana — APPROVED** [review]

File: docs/DeveloperGuide.md (hunk 4 above)
Comment (Chongjx) [reply docs/DeveloperGuide.md:92]: https://nus-cs2113-ay2021s1.github.io/website/schedule/week10/topics.html#w10-2-sequence-diagrams-intermediate-level

Comment (R-Ramana) [reply docs/DeveloperGuide.md:92]: Not the content haha, line 91 just ends with "due to the limitation of".

Comment (Chongjx) [reply docs/DeveloperGuide.md:92]: Noted.

**Pull request merged**